software/bytestash.yml

```
@@ -0,0 +1,14 @@
+name: "ByteStash"
+website_url: "https://github.com/jordan-dalby/ByteStash"
+source_code_url: "https://github.com/jordan-dalby/ByteStash"
+description: "Self-hosted pastebin and file storage service with a simple web interface and Docker support. Supports syntax highlighting, optional user authentication and public sharing."
+licenses:
+  - MIT
```

Improved description and fixed Licenses

```suggestion
description: "Pastebin and file storage service with a simple web interface. Supports syntax highlighting, optional user authentication and public sharing."
licenses:
  - GPL-3.0
```